client.lua

```diff
@@ -1,4 +1,16 @@
+if (Config.FRAMEWORK == 'qb') then
+    FRAMEWORK = Config.GET_CORE
+elseif (Config.FRAMEWORK == 'esx') then
+    FRAMEWORK = exports["es_extended"]:getSharedObject()
+elseif (Config.FRAMEWORK == 'esx-old') then
+    TriggerEvent('esx:getSharedObject', function(obj) FRAMEWORK = obj end)
+else
+    -- put in custom logic to grab framework and delete print code underneath
+    print('^6[^3dream-postal^6]^0 Unsupported Framework detected!')
+end
```

Comment:
We already grab framework in client_util so this code is not needed. With that being said is client_util loaded in first? If not then we must change order in fxmanifest.lua